sdk/spring/azure-spring-boot/README.md

```diff
@@ -61,7 +61,7 @@ The following section provides sample projects illustrating how to use the Azure
 - [Azure Active Directory for WebApi](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server)
 - [Azure Active Directory for WebApp](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp)
 - [Azure Active Directory for Resource Server](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-spring-security-resource-server)
-- [Azure Active Directory for WebApi Obo](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo)
+- [Azure Active Directory for Resource Server with Obo Clients](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo)
```

Comment:
Suggestions:
1. Rename `WebApi` to `Resource Server By Filter (depreacted)`.
2. Rename `azure-spring-boot-sample-active-directory-resource-server` to `azure-spring-boot-sample-active-directory-resource-server-with-filter` (Add `-filter`)
3. Rename `WebApp` to `Web Application`.
4. Rename `azure-spring-boot-sample-active-directory-spring-security-resource-server` to `azure-spring-boot-sample-active-directory-resource-server` (Delete `-spring-security`)
5. Change order to `Web Application` -> `Resource Server` -> `Resource Server with Obo Clients` -> `Resource Server By Filter (depreacted)`